SEImplementation/SEImplementation/Configuration/DetectionImageConfig.h

```diff
@@ -62,10 +63,16 @@ class DetectionImageConfig : public Euclid::Configuration::Configuration {
   // we still need to know what it was to adjust the weight map
   double getOriginalFluxScale() const { return m_flux_scale; }
 
+  // Get the detection image source
```

Comment:
I don't really like that we are making a dependency to FitsImageSource here. I think it's not completely unreasonable to imagine adding support for another format in the future or we could imagine a detection image that doesn't come directly from a file.

Maybe we could have an abstract class for sources that can return or generate FITS compatible headers? Or maybe just return a generic ImageSource?